client/src/components/ReadMore.jsx

```diff
@@ -0,0 +1,102 @@
+import React, {Component} from 'react';
```

Comment:
Later on, you may be able to refactor some code so that your ReadMore and ShowHalfText can share some lines of code instead of repeating.